rasa/core/policies/flow_policy.py

```diff
@@ -372,202 +514,189 @@ def _is_step_completed(
         """Check if a step is completed."""
         if isinstance(step, QuestionFlowStep):
             return tracker.get_slot(step.question) is not None
-        elif isinstance(step, LinkFlowStep):
-            # The flow step can't be completed this way, it get's completed
-            # when the linked flow wraps up and returns to this flow
-            return False
         else:
             return True
 
-    def _get_current_flow(self) -> Optional[Flow]:
-        """Get the current flow.
+    def start_flow_if_possible(
```

Comment:
as said above, better name might be `consider_starting_new_flow`. `start_flow_if_possible` gives me the impression there is already a flow I know I want to start and I am checking whether the conditions are met. But actually the method tries to answer the question whether some new flow should be started.